libraries/botbuilder-ai/src/qnaMaker.ts

```diff
@@ -193,18 +193,25 @@ export class QnAMaker {
      * @param top (Optional) number of answers to return. Defaults to a value of `1`.
```

Comment:
These @param comments are not in sync with the changed code below. Please make them match.